### DISCUSSION TITLE
Cartoonito Uk

### DISCUSSION THREAD

**Discussion by cohendjdjd** [iptv-org/community#373] [Show and tell]
### Channel ID

CartoonitoUK.uk

### Stream URL

https://list.iptvcat.com/my_list/s/2d8b4d9b4951dfce66872f9a063313db.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Legal link to stream?

**Comment by cohendjdjd** [comment]
https://list.iptvcat.com/my_list/s/2d8b4d9b4951dfce66872f9a063313db.m3u8 

tell Me If This Link Works

**Comment by AntiPontifex** [comment]
Link does not work: I get a screen that says login error.

**Comment by cohendjdjd** [comment]
Can’t find any streams do you know how to find them

**Comment by freearhey** [comment]
https://list.iptvcat.com/my_list/s/2d8b4d9b4951dfce66872f9a063313db.m3u8 (offline)
<img width="541" alt="image" src="https://user-images.githubusercontent.com/7253922/232254229-beb5c48b-1c4d-4136-9cc1-a1a7a968c9d7.png">
